Lib/test/test_opcache.py

```diff
@@ -576,6 +576,7 @@ class TestRacesDoNotCrash(TestBase):
     # Careful with these. Bigger numbers have a higher chance of catching bugs,
     # but you can also burn through a *ton* of type/dict/function versions:
     ITEMS = 1000
+    SMALL_ITEMS = 100
```

Comment:
It might be worth investigating this further. I'm surprised there's a large slowdown in this PR, but not in the earlier version. What's the relevant difference?

Reply:
I timed the most recent version of this PR vs "main", using `SMALL_ITEMS = 1000`.  It seems the slowdown is not too bad now.  I've left it at 100 but we could revert to 1000 if we want to.

I wasn't able to exactly pin-point the reason for the huge slowdown before.  Based on "samply" profiling, it looked like there was a lot of contention for the `TYPE_LOCK` mutex and the STM mutex.  Quite a lot of time was spent in `_Py_yield()`.   That's kind of expected since the two writers are both trying to acquire both of those.

Reply:
I reverted this change to test_opcache.